filetests/isa/x86/shuffle-compile.clif

```diff
@@ -0,0 +1,31 @@
+test compile
```

Comment:
Could this be a test legalizer test, instead? Or does it depend on other compiler passes?